app/controllers/api/account.php

```diff
@@ -3648,12 +3648,119 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
     ->label('sdk.offline.key', 'current')
     ->param('mfa', null, new Boolean(), 'Enable or disable MFA.')
     ->inject('requestTimestamp')
+    ->inject('request')
     ->inject('response')
     ->inject('user')
+    ->inject('project')
     ->inject('session')
+    ->inject('locale')
     ->inject('dbForProject')
     ->inject('queueForEvents')
-    ->action(function (bool $mfa, ?\DateTime $requestTimestamp, Response $response, Document $user, Document $session, Database $dbForProject, Event $queueForEvents) {
+    ->inject('queueForMails')
+    ->action(function (bool $mfa, ?\DateTime $requestTimestamp, Request $request, Response $response, Document $user, Document $project, Document $session, Locale $locale, Database $dbForProject, Event $queueForEvents, Mail $queueForMails) {
+        // If MFA Changes then we need to send a email
+        if ($mfa !== $user->getAttribute('mfa')) {
+            $domain = $request->getHostname();
+            $protocol = $request->getProtocol();
+
+            $redirect = $protocol . '://' . $domain . '/console/account';
+            $type = $mfa ? 'mfaEnabled' : 'mfaDisabled';
+
+            $subject = $locale->getText("emails.{$type}.subject");
+            $customTemplate = $project->getAttribute('templates', [])["email.{$type}-" . $locale->default] ?? [];
+
+            // Appwrite Theming for Console
+            if ($project->getInternalId() === 'console') {
+                $message = Template::fromFile(__DIR__ . '/../../config/locale/templates/email-base-styled.tpl');
+                $body = Template::fromFile(__DIR__ . '/../../config/locale/templates/email-mfa-altered-console.tpl');
+
+                $buttonText = $locale->getText("emails.{$type}.buttonText", 'console-'.$locale->default) ?? $locale->getText("emails.{$type}.buttonText");
```

Comment:
Ah I see, I got confused because we have $default and $this->default, that second one should probably be `$this->defaultLanguage` to remove confusion or even `$this->currentLanguage`